index.html

```diff
@@ -35,7 +38,7 @@
         <form action="LOREM IPSOM" method="post">
           <div class="form-item">
             <div class="form-item-label">Date</div>
```

Comment:
Trying to choose a date while the intro is being shown seems to be having messy flow. Try digging in a little more to find how to stop the 'intro' when the user clicks on the input field and resume it again after he has chosen a value. Ditto this for all input fields.